index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport"
+		  content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Document</title>
+	<script type="text/javascript" src="templater.js"></script>
+</head>
+<body>
+	<bootstrap_button>Some Text</bootstrap_button>
```

Comment:
Please add more than one instances of custom tag in your html file, just to check that it works when there are more than one `bootstrap_button` tag in page